bciers/libs/shared/actions/.eslintrc.json

```diff
@@ -1,6 +1,10 @@
 {
```

Comment:
Could we cut down on some of the `.eslintrc.json`s and use the main one in bciers instead? `libs/shared` has three and I'm guessing that any different rules they might have ended up with weren't intentional

Reply:
Same as the above comment, generated by nx and good to leave in to tweak in the future. Though for the most part my opinion is that we should keep most of the entire repo on the same rules but with minor tweaks in the future or disabling new rules for projects in maintenance eg registration1.

Reply:
Yes, it would be good to use the same rules when we can